layouts/index.html

```diff
@@ -52,6 +52,22 @@
         term.echo("")
     }
 
+    function certificates () {
+          term.echo("");
+          {{ with .Site.Params.certificates }}
+              {{ range . }}
+			      term.echo("[[b;{{ .color | default "grey" }};]{{ .certName }}]");
+                  term.echo("{{ .company }}");
```

Comment:
Let's make this one optional. The certifying company is often irrelevant and the certificate name by itself is enough. 

For example AWS Certificates are provided by AWS themselves therefore the name of the certification by itself is enough. 

Use same method as for the description. 

```
{{ if .company }}
    term.echo("{{ .company }}");
{{ end }}
```

Reply:
Another good idea!